techniques/systemSettings/misc/clockConfiguration/3.0/changelog

```diff
@@ -4,3 +4,7 @@
  -- Benoît Peccatte <benoit.peccatte@normation.com> Fri Oct 17 14:10:43 CEST 2014
   * Version 3.0
   ** Use rudder_common_report instead of reports:
+ -- Janos Mattyasovszky <mail@matya.hu> Sun Jan 31 01:04:00 CET 2016
+  * Version 3.0
```

Comment:
I think there's been an error there. Adding new stuff to a technique should make it a new version (3.1)